app/gui2/src/components/widgets/Dropdown.vue

```diff
@@ -0,0 +1,111 @@
+<script setup lang="ts">
+import SortAscendingIcon from '@/assets/icons/sort_ascending.svg'
+import SortDescendingIcon from '@/assets/icons/sort_descending.svg'
+import SortIcon from '@/assets/icons/sort.svg'
+import { computed, ref } from 'vue'
+
+enum SortDirection {
+  none = 'none',
+  ascending = 'ascending',
+  descending = 'descending',
+}
+
+const props = defineProps<{ color: string; selectedValue: string | null; values: string[] }>()
+const emit = defineEmits<{ click: [index: number] }>()
+
+const sortDirection = ref<SortDirection>(SortDirection.none)
+
+const sortedValuesAndIndices = computed(() => {
+  const valuesAndIndices = props.values.map<[value: string, index: number]>((value, index) => [
+    value,
+    index,
+  ])
+  switch (sortDirection.value) {
+    case SortDirection.none: {
+      return valuesAndIndices
+    }
+    case SortDirection.ascending: {
+      return valuesAndIndices.sort((a, b) => (a[0] > b[0] ? 1 : a[0] < b[0] ? -1 : 0))
+    }
+    case SortDirection.descending: {
+      return valuesAndIndices.sort((a, b) => (a[0] > b[0] ? -1 : a[0] < b[0] ? 1 : 0))
+    }
+  }
+})
+
+const ICON_LOOKUP: Record<SortDirection, string> = {
+  [SortDirection.none]: SortIcon,
+  [SortDirection.ascending]: SortAscendingIcon,
+  [SortDirection.descending]: SortDescendingIcon,
+}
+
+const NEXT_SORT_DIRECTION: Record<SortDirection, SortDirection> = {
+  [SortDirection.none]: SortDirection.ascending,
+  [SortDirection.ascending]: SortDirection.descending,
+  [SortDirection.descending]: SortDirection.none,
+}
+</script>
+
+<template>
+  <div :tabindex="-1" class="Dropdown" @wheel.stop>
+    <ul class="list" :style="{ background: color }">
+      <template v-for="[value, index] in sortedValuesAndIndices" :key="value">
+        <li v-if="value === selectedValue">
+          <div class="selected-item"><span v-text="value"></span></div>
+        </li>
+        <li v-else class="selectable-item button" @click="emit('click', index)">
+          <span v-text="value"></span>
+        </li>
+      </template>
+      <img
+        class="sort button"
+        :src="ICON_LOOKUP[sortDirection]"
+        @click="sortDirection = NEXT_SORT_DIRECTION[sortDirection]"
+      />
+    </ul>
+  </div>
+</template>
+
+<style scoped>
+.Dropdown {
+  position: absolute;
+  user-select: none;
+  overflow: auto;
+  clip-path: inset(0 round 8px);
+  width: min-content;
+  top: 100%;
+  margin-top: 4px;
+  height: 136px;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
+
+.Dropdown::-webkit-scrollbar {
+  display: none;
```

Comment:
We need a scrollbar. We can use native ones, but it should be styled to mimic the one on the design, at least as close as CSS will allow. We might eventually move on to custom scrollbar components, but we have to have something functional until that happens.

Reply:
i don't think there are any scrollbars anywhere in the design actually - but can definitely add one

Reply:
When in doubt, look at what's implemented in current IDE. We have scrollbars. The design simply missed them/didn't show them in shown state. This could also be raised with @wdanilo to make sure it's visible in figma in at least one place.

Reply:
i don't get scrollbars in ide unfortunately :(
unless it's a fairly recent feature? or unless we're supposed to use the same scrollbars as in the CB

![image](https://github.com/enso-org/enso/assets/4046547/f825a3d9-266b-4f9c-ba97-bb5422087a4a)

Reply:
I guess the dropdowns are allowed to be quite long before they show the scrollbar. But it is the same component as in the CB indeed.